models/User.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose')
+
+const UserSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  password: {
+    type: String,
+    required: true,
+    minlength: 6
+  },
+  avatar: {
```

Comment:
Make the user upload a profile photo.

Reply:
The type is string, and a photo is generally a set of bytes. Does this mean we'll be base64 encoding the image?

Reply:
I will remove this from the User model and put it in Profile. This implementation is a bit more complex I wanna wait until I reach the frontend, probably I'll be using a `multer` but for now it will be a temporary profile placeholder.